packages/@aws-cdk/aws-eks/README.md

```diff
@@ -15,20 +15,173 @@
 ---
 <!--END STABILITY BANNER-->
 
-This construct library allows you to define and create [Amazon Elastic Container Service for Kubernetes (EKS)](https://aws.amazon.com/eks/) clusters programmatically.
+This construct library allows you to define [Amazon Elastic Container Service
+for Kubernetes (EKS)](https://aws.amazon.com/eks/) clusters programmatically.
 
-### Example
+This library also supports programmatically defining Kubernetes resource
+manifests within EKS clusters.
 
-The following example shows how to start an EKS cluster and how to
-add worker nodes to it:
+This example defines an Amazon EKS cluster with a single pod:
 
-[starting a cluster example](test/integ.eks-cluster.lit.ts)
+```ts
+const cluster = new eks.Cluster(this, 'hello-eks', { vpc });
```

Comment:
Can we take reasonable defaults and create the cluster as shown?

```
const cluster = new eks.Cluster(this, 'hello-eks’);
```

This means creating a default VPC, creating a data plane with a reasonable, say 2, worker nodes. You can also pick a reasonable type, say `m5.large`. Make this experience similar to `eksctl create cluster`.

Reply:
Will address in a subsequent PR: https://github.com/aws/aws-cdk/issues/3541